codalab/lib/codalab_manager.py

```diff
@@ -326,11 +326,20 @@ def session(self):
     @cached
     def default_user_info(self):
         info = self.config['server'].get(
-            'default_user_info', {'time_quota': '1y', 'disk_quota': '1t', 'parallel_run_quota': 3}
+            'default_user_info',
+            {
+                'time_quota': '1y',
```

Comment:
Shouldn't time_quota and disk_quota be lowered (so that users without edu emails have less quota)?

Reply:
Yeah it should. But actually if we set environmental variables, these parameters here will not be used.